includes/meta.php

```diff
@@ -38,24 +38,43 @@ function sticky_post_for_term( $post_id, $term_id ) {
 }
 
```

Comment:
does the `$term_id` need to be set to 0 in the function call (line 21) or check for an `empty()` before proceeding?

Reply:
comparing `$new_meta` and `$existing_meta` when a new tag is passed, the `array_map() intval` it just checks against a zero. it works (because the arrays still don't match) but wanted to note that.